src/main/java/dev/slimevr/vr/processor/skeleton/HumanSkeleton.java

```diff
@@ -328,124 +328,26 @@ public static float interpolateRadians(float factor, float start, float end) {
 		return normalizeRad(val);
 	}
 
+	/** rebuilds the `currentBoneInfo` list */
 	protected void resetBones() {
 		currentBoneInfo.clear();
-		// #region Assemble skeleton from hmd to hip
-		currentBoneInfo.add(new BoneInfo(BoneType.HEAD, headNode));
```

Comment:
I see I had some typos while copy-pasting, but why did you remove most of the bones? If you want to filter them, you can do it on the client side.

Reply:
resolve when ready